plugins/login-plugin/src/Components/UIComponents/Storefront/StoreFrontTv/EventDateTitle.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { StyleSheet } from "react-native";
+import Label from "../../Label";
+import { mapKeyToStyle } from "../../../../Utils/Customization";
+import PropTypes from "prop-types";
+
+const EventDateTitle = (props) => {
+  const {
+    screenStyles,
+    payload: { extensions = {} },
+  } = props;
+
+  const timestamp = extensions?.event_date;
+  if (!timestamp) {
+    return null;
+  }
+
+  const fontStyles = mapKeyToStyle("event_schedule_text", screenStyles);
+
+  const styles = StyleSheet.create({
+    container: {
+      marginTop: 30,
+    },
+    text: {
+      ...fontStyles,
+      alignSelf: "center",
+      textAlign: "center",
+    },
+  });
+
+  var date = new Date(timestamp * 1000);
```

Comment:
Don't use var, please. Use cont instead.
Ther reason not to use `var` is that it's a function scoped variable and `const` is block scoped. This can lead to bugs as we can accidentally overwrite a variable. 

It's considered a good practice to just use `const` and `let`